docs/src/bpmn-workflows/message-events.md

```diff
@@ -11,7 +11,11 @@ A message can be referenced by one or more message events. It holds the informat
 * the name of the message
 * the correlation key
 
+<<<<<<< HEAD
```

Comment:
I think this might be left overs from a merge conflict?

Reply:
Ach, you're right. Unstaged it. Thanks.